lib/hypervisor/hv_kvm/monitor.py

```diff
@@ -771,6 +771,14 @@ def StartPostcopyMigration(self):
 
     self.Execute("migrate-start-postcopy")
 
+  @_ensure_connection
+  def GetFastCpuInformation(self):
```

Comment:
I'd leave "fast" out of the name here.

Reply:
Yes, that's probably not relevant outside the monitoring class. Will fix.